render_files.py

```diff
@@ -81,15 +81,16 @@ def upload_to_s3(s3_endpoint_url, bucket, aws_access_key_id, aws_secret_access_k
             upload_to_aws(s3_client, file_path, bucket, s3_file_name)
 
 
-def debug_print_upload_to_s3(install_dir):
-    prefix = "dummy_cluster_id"
+def copy_to_local_storage(work_dir, install_dir, cluster_id):
+    os.makedirs(os.path.join(work_dir, cluster_id), exist_ok=True)
     for root, _, files in os.walk(install_dir):
         for file_name in files:
             file_path = os.path.join(root, file_name)
             if file_name == "kubeconfig":
                 file_name = "kubeconfig-noingress"
```

Comment:
It would be great if we could have a generic function that contained the os.walk, changing the kubeconfig file_name, etc. and then differed only in copying to S3 or file system.